CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -1051,7 +1051,7 @@ void __CFInitialize(void) {
         #if TARGET_OS_LINUX
         #define __CFSwiftGetBaseClass _TF10Foundation21__CFSwiftGetBaseClassFT_PMPs9AnyObject_
         #elif TARGET_OS_MAC
-        #define __CFSwiftGetBaseClass _TF15SwiftFoundation21__CFSwiftGetBaseClassFT_PMPs9AnyObject_
+        #define __CFSwiftGetBaseClass _TF18PortableFoundation21__CFSwiftGetBaseClassFT_PMPs9AnyObject_
```

Comment:
This seems like a change just to change the name right? Again these changes seem to be a fully different set of thoughts here (irregardless of what the name may infer). Personally I think that it would be nice if we could somehow unify both targets to be the same thing: but that comes at some complexity.